app/views/workshop_invitation_mailer/invite_student.html.haml

```diff
@@ -13,12 +13,16 @@
               %td
                 %h3 Hi #{@member.name},
                 %p.lead
-                  We’d love to have you at our next #{@workshop.chapter.name} workshop. RSVP now to secure a spot, some light bites and time with a dedicated coach.
-                %p
-                  As part of our workshop, you'll get to work through any of our #{link_to 'tutorials', 'http://codebar.github.io/tutorials' } or receive guidance on your personal project. We believe that everyone should be entitled to free learning and our community has #{ link_to 'a lot of devoted developers', full_url_for(coaches_url)} who help out as coaches.
-                %p You will also get the opportunity to meet other people interested in coding and collaborate with them.
-                %p We’re looking forward to having you with us. See you there!
-                %p Please note: We do not accept any RSVPs over email.
+                  We’d love to see you at the next #{@workshop.chapter.name} workshop! Don’t miss this chance to secure your spot, enjoy some light refreshments, and work one-on-one with a dedicated coach.
```

Comment:
```suggestion
                  We’d love to see you at the next #{@workshop.chapter.name} workshop! Don’t miss this chance to secure your spot, enjoy some light refreshments, and work one-on-one or in a small group with a dedicated coach.
```
@KimberleyCook do we want to say one-on-one here? I can't always guarantee a student will work one on one with a coach. I'd say work one-on-one or in a small group with a coach.